### ISSUE TITLE
warning v-alert text color

### ISSUE THREAD

**Issue by Christilut** [vuetifyjs/vuetify#1689]
`v-alert` with `warning` prop has white text on orange background which is not super easy to read. 
I guess this is in the material design but MD states that legibility needs to have a contrast ratio of higher than 3 but it [fails](http://leaverou.github.io/contrast-ratio/#white-on-rgb%28255%2C%20193%2C%207%29)

**Comment by johnleider** [comment]
When creating an issue you should be provided with boilerplate for the ticket. This is important for myself and others to properly address your issue. Please reopen this with the required information, thank you.

**Comment by johnleider** [comment]
When creating an issue you should be provided with boilerplate for the ticket. This is important for myself and others to properly address your issue. Please re-open a new issue with the required information, thank you.
